### PR TITLE
Association

### PR DESCRIPTION
**Pull request by ciraganenicole**
In this pull request I:

- Create Classroom class
- Create Book class
- Create Rental class
- Add association

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+require './book'
+
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.visits << self
```
Comment (keneogu) [inline rental.rb]: Can we make this to be `person.rentals << self` since we working with rentals.

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    @book = book
+    book.visits << self
```
Comment (keneogu) [inline rental.rb]: Can we make this to be `book.rentals << self` since we working with rentals. The rental is adding itself to the @book. Nice one though.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @ciraganenicole, 

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history
- Good PR description
- 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi **@ciraganenicole,** 

Thank you for implementing all changes requested by previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

cong


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**